### ISSUE TITLE
Point release required to include PR#1641 on cucumber:main

### ISSUE THREAD

**Issue by diabolo** [cucumber/cucumber-ruby#1677]
Cucumber 8.0.0 (the current release at time of writing) crashes if the environment it is running in has  CUCUMBER_COLORS set. PR#1641 fixed this and was merged into cucumber:main at the end of May. This needs to be released.

**Comment by mattwynne** [comment]
Thanks for bringing this up @diabolo. Since [Aurélien left](https://cucumber.io/blog/open-source/au-revoir-aurelien/) earlier this year we've not had anyone really focused on cucumber-ruby to make releases etc.

**Comment by mattwynne** [comment]
Waiting on https://github.com/cucumber/cucumber-ruby/pull/1686 - the dependencies are out of date and there's a bit of a rabbit hole.

**Comment by luke-hill** [comment]
Released in v9 & v9.0.1 should now be used
